sbt-plugin/src/sbt-test/gen-scala-server/10-scalapb-validate/test

```diff
@@ -0,0 +1 @@
+> compile
```

Comment:
is there more we can test beyond 'it compiles'?

Reply:
good point, I'll add a basic `Main`

Reply:
First commit amended. I wanted to go even further and verify/show that [`validate_at_construction`](https://github.com/scalapb/scalapb-validate/blob/6de254da20b8e82d3334366705067feea6ece6c9/core/src/main/protobuf/scalapb/validate.proto#L26) could be enabled so that clients sending invalid payloads would automatically get an error (maybe with a cutom exception handler) without any specific custom code on the server implementation, but I ran into https://github.com/scalapb/scalapb-validate/issues/70#issuecomment-779699193 so that will be for the next round.

Reply:
> maybe with a cutom exception handler to map `scalapb.validate.ValidationFailure` to `INVALID_ARGUMENT`)

that might even be reasonable to make part of the default handler

Reply:
> that might even be reasonable to make part of the default handler

I got the type wrong, it's [`scalapb.validate.ValidationException`](https://github.com/scalapb/scalapb-validate/blob/a39f93814523b7c74d97d6fba20ccfbfa386621d/core/src/main/scala/scalapb/validate/Validator.scala#L100-L101), which extends `com.google.protobuf.InvalidProtocolBufferException`. Would it make sense to map `InvalidProtocolBufferException` to `INVALID_ARGUMENT` ?

Reply:
that makes sense to me

Reply:
that sounds entirely reasonable!